ci/gpu/build.sh

```diff
@@ -49,43 +46,26 @@ nvidia-smi
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
+
 conda info
 conda config --show-sources
 conda list --show-channel-urls
 
-# Fixing Numpy version to avoid RuntimeWarning: numpy.ufunc size changed, may
-# indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject.
-# Also installing cucim in order to test GDS spilling
-gpuci_mamba_retry install "cudatoolkit=$CUDA_REL" \
-              "cudf=${MINOR_VERSION}" "dask-cudf=${MINOR_VERSION}" \
-              "ucx-py=${UCXPY_VERSION}" "ucx-proc=*=gpu" \
-              "rapids-build-env=$MINOR_VERSION.*" \
-              "cucim"
-
+# Installing cucim in order to test GDS spilling
 # Pin pytest-asyncio because latest versions modify the default asyncio
 # `event_loop_policy`. See https://github.com/dask/distributed/pull/4212 .
-gpuci_mamba_retry install "pytest-asyncio=<0.14.0"
-
-# https://docs.rapids.ai/maintainers/depmgmt/
-# gpuci_mamba_retry remove -f rapids-build-env
-# gpuci_mamba_retry install "your-pkg=1.0.0"
-
-conda info
-conda config --show-sources
-conda list --show-channel-urls
-
-# Install the main version of dask and distributed
-if [[ "${INSTALL_DASK_MASTER}" == 1 ]]; then
-    gpuci_logger "pip install git+https://github.com/dask/distributed.git@main --upgrade"
-    pip install "git+https://github.com/dask/distributed.git@main" --upgrade
-    gpuci_logger "pip install git+https://github.com/dask/dask.git@main --upgrade"
-    pip install "git+https://github.com/dask/dask.git@main" --upgrade
-fi
```

Comment:
quick question. was this block removed since `dask` is installed via `dask-cudf`?

Reply:
But we always want to install Dask from the main branch in CI, are we doing that via Dask conda nightly channel?

Reply:
Doesn't seem like it from the gpuCI runs - should we consider taking an approach similar to [cuDF's](https://github.com/rapidsai/cudf/blob/1519108adcdb520ff4b41849d7dc6cf61330c2fc/ci/gpu/build.sh#L92-L99) here?

Reply:
This commit should solve this: https://github.com/rapidsai/dask-cuda/compare/b0024dbbbeaf659aac43118443a2aac4033e9615..601f9a79be30df766587886c748858413e26861e

Reply:
We would prefer to keep the `INSTALL_DASK_MAIN` environment variable as it was before, and as is in the cuDF CI script that Charles mentioned above. Every release during code freeze we just flip that variable to use the release version of Dask and then flip it over again on the new branch.